### ISSUE TITLE
react intl v3

### ISSUE THREAD

**Issue by zabojad** [kLabz/haxe-react-intl#1]
The react-intl v3 comes with an enhanced API (new functions + some deprecations).

Should we keep the current version in a v2 branch and start a v3 support in the master branch?

I can PR something once I'll have updated my current project to react intl v3...

**Comment by kLabz** [comment]
Yep, a v2 branch could be created from current master for potential future updates; additionally, a 2.0.0 haxelib version should be released with the same content as 0.2.0.

v3 compatibility should be released as 3.x.x

(note that I don't use react-intl in any project anymore so don't expect much from me for v3 compatibility)
